server/api.go

```diff
@@ -853,3 +855,54 @@ func (p *Plugin) updateSettings(w http.ResponseWriter, r *http.Request, userID s
 	resp, _ := json.Marshal(info.Settings)
 	w.Write(resp)
 }
+
+func (p *Plugin) getIssueByNumber(w http.ResponseWriter, r *http.Request, userID string) {
+
+	ctx := context.Background()
+
+	var githubClient *github.Client
+	owner := r.FormValue("owner")
+	repo := r.FormValue("repo")
+	number := r.FormValue("number")
+	numberInt, _ := strconv.Atoi(number)
```

Comment:
Given that this is a user input, the error should be properly checked and `Debug` logged.

Reply:
Makes sense!